hibernate-reactive-core/src/test/java/org/hibernate/reactive/BasicTypesAndCallbacksTest.java

```diff
@@ -1,359 +0,0 @@
-							context.assertEquals( basic.string, basik.string);
-							context.assertEquals( basic.decimal.floatValue(), basik.decimal.floatValue());
-							context.assertEquals( basic.integer, basik.integer);
-							context.assertTrue( Arrays.equals(basic.bytes, basik.bytes) );
-							context.assertEquals( basic.timeZone, basik.timeZone);
-							context.assertEquals( basic.cover, basik.cover);
-							context.assertEquals( basic.binInteger.floatValue(), basik.binInteger.floatValue());
-							context.assertEquals( basic.date, basik.date );
-							context.assertEquals( basic.localDate, basik.localDate );
-							context.assertEquals( basic.localDateTime, basik.localDateTime );
-							context.assertEquals( basic.localTime.truncatedTo(ChronoUnit.MINUTES),
-									basik.localTime.truncatedTo(ChronoUnit.MINUTES) );
-							context.assertTrue( basic.thing instanceof String[] );
-							context.assertTrue( Objects.deepEquals(basic.thing, basik.thing) );
-							context.assertEquals( basic.embed, basik.embed );
-							context.assertEquals( basic.version, 0 );
-
-							basic.string = "Goodbye";
-							basic.cover = Cover.soft;
-							basic.parent = new Basic("New Parent");
-							return s2.persist(basic.parent)
-									.thenCompose( v -> s2.flush() )
-									.thenAccept(v -> {
-										context.assertNotNull( basic );
-										context.assertTrue( basic.postUpdated && basic.preUpdated );
-										context.assertFalse( basic.postPersisted && basic.prePersisted );
-										context.assertTrue( basic.parent.postPersisted && basic.parent.prePersisted );
-										context.assertEquals( basic.version, 1 );
-									});
-						}))
-				.thenCompose(v -> openSession())
-				.thenCompose(s3 ->
-					s3.find( Basic.class, basik.getId() )
-						.thenCompose( basic -> {
-							context.assertFalse( basic.postUpdated && basic.preUpdated );
-							context.assertFalse( basic.postPersisted && basic.prePersisted );
-							context.assertEquals( basic.version, 1 );
-							context.assertEquals( basic.string, "Goodbye");
-							return s3.createQuery("update Basic set string = 'Goodnight'").executeUpdate()
-									.thenCompose(v -> s3.refresh(basic))
-									.thenAccept(v -> context.assertEquals(basic.getString(), "Goodnight"))
-									.thenCompose(v -> {
-										basik.version = basic.version;
-										return s3.merge(basik);
-									})
-									.thenAccept( b -> context.assertEquals( b.string, "Hello World") )
-									.thenCompose(v -> s3.remove(basic))
-									.thenAccept(v -> context.assertTrue( !basic.postRemoved && basic.preRemoved ) )
-									.thenCompose(v -> s3.flush())
-									.thenAccept(v -> context.assertTrue( basic.postRemoved && basic.preRemoved ) );
-						}))
-				.thenCompose(v -> openSession())
-				.thenCompose(s4 ->
-						s4.find( Basic.class, basik.getId() )
-							.thenAccept(context::assertNull))
-		);
-	}
-
-	enum Cover { hard, soft }
-
-	@Embeddable
-	static class Embed {
```

Comment:
I don't see the tests for enum and embedded types.

Reply:
They are all in: BasicTypesAndCallbacksForAllDBsTest: https://github.com/hibernate/hibernate-reactive/blob/abb3ecd984df9847a7b8c52ac897248f24c25039/hibernate-reactive-core/src/test/java/org/hibernate/reactive/types/BasicTypesAndCallbacksForAllDBsTest.java#L245

Reply:
https://github.com/hibernate/hibernate-reactive/blob/abb3ecd984df9847a7b8c52ac897248f24c25039/hibernate-reactive-core/src/test/java/org/hibernate/reactive/types/BasicTypesAndCallbacksForAllDBsTest.java#L231

Reply:
OK sorry, I'm not sure how I missed that file. Probably because was trying to use the github iPad client and not used to it.

Reply:
No problem, normally I would keep the history about renaming the file but I think this time I messed up somewhere and didn't want to spend more time to rebase everything.